src/ray/protobuf/core_worker.proto

```diff
@@ -67,6 +67,9 @@ message ActorHandle {
 
   // The max number of pending actor calls.
   int32 max_pending_calls = 13;
+
+  // Whether or not an actor is an async actor.
+  bool is_asyncio = 14;
```

Comment:
Is this used?

Reply:
nope good catch! (I tried using it to only support async actor, but the current protocol works for everything anyway).